components/automate-ui/src/styles/mixins/_mat-theming.scss

```diff
@@ -168,9 +168,38 @@ $backdrop-animation-timing-function: cubic-bezier(0.25, 0.8, 0.25, 1) !default;
 //
 // @param target Which kind of high contrast setting to target. Defaults to `active`, can be
 //    `white-on-black` or `black-on-white`.
-@mixin cdk-high-contrast($target: active) {
-  @media (-ms-high-contrast: $target) {
-    @content;
+@mixin cdk-high-contrast($target: active, $encapsulation: 'any') {
```

Comment:
Updated this file to the latest version from angular material. (This file is an excerpt from _theming.scss and the version we had failed to compile.)

Reply:
That was @scottopherson's good work. 😁

Reply:
Great work @scottopherson !